src/main/java/io/swagger/configuration/ScheduledTasks.java

```diff
@@ -0,0 +1,29 @@
+package io.swagger.configuration;
+
+import io.swagger.model.Account;
+import io.swagger.service.AccountService;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@Component
+public class ScheduledTasks {
+
+    private static final Logger logger = Logger.getLogger(ScheduledTasks.class.getName());
+    private final AccountService accountService;
+
+    public ScheduledTasks(AccountService accountService) {
+        this.accountService = accountService;
+    }
+
+    @Scheduled(cron = "0 1 * * * ?")
+    public void resetAccountDayLimit(){
+        logger.log(Level.INFO,"Resetting day limit...");
+        for (Account account : accountService.getAllAccounts()){
```

Comment:
probeer zo min mogelijk for-loops te gebruiken. Probeer het even uit met accountService.getAllAccounts().foreach .
Dit maakt gebruik van java Streams, wat veel meer geoptimaliseerd is zo gauw er grote aantallen requests in je backend komen. Probeer het zelf even uit, als je er niet uit komt kijk ik aankomende zondag in de meeting met je mee

Reply:
https://www.baeldung.com/foreach-java#:~:text=We%20use%20forEach%20to%20iterate,with%20a%20single%20abstract%20method).